src/subscriptions.js

```diff
@@ -85,7 +85,7 @@ export default tracker => ({ dispatch }) => next => action => {
       });
     };
 
-    runContextual(start);
+    run(start);
```

Comment:
[above comment](https://github.com/samybob1/meteor-redux-middlewares/pull/15/files#r94862359)

could just be `setTimeout`